Lib/fontTools/feaLib/parser.py

```diff
@@ -924,23 +924,25 @@ def parse_substitute_(self):
             )
 
         # GSUB lookup type 2: Multiple substitution.
-        # Format: "substitute f_f_i by f f i;"
-        if (
-            not reverse
-            and len(old) == 1
-            and len(old[0].glyphSet()) == 1
-            and len(new) > 1
-            and max([len(n.glyphSet()) for n in new]) == 1
-            and num_lookups == 0
-        ):
+        # Format A: "substitute f_f_i by f f i;"
+        # Format B: "substitute [f_i f_l] by [f f] [i l];"
+        # Format C: "substitute [f_i f_l] by f [i l];"
```

Comment:
where do these A, B and C come from? I don't see them listed in http://adobe-type-tools.github.io/afdko/OpenTypeFeatureFileSpecification.html#5.b

Reply:
I pulled the numbers out of my hat, as I was planning to propose the additional syntax to AFDKO. I’m goingto add a link to Glyphs documentation there, and name them GlyphsApp format 1 and 2.